### ISSUE TITLE
Feature Request !!

### ISSUE THREAD

**Issue by kmt29** [tameemsafi/typewriterjs#64]
I think the lib is really small, portable yet so powerful. But i think it will even be more powerful if you guys add a feature that could clear the loop queue.

**Comment by markerio** [comment]
I agree with @kmt29, it would be great if they added a feature to stop at least.

**Comment by llibdude** [comment]
As a work around i'm finding this is generally working for me
```javascript
typewriterRef.current.state.eventQueue = []
```

In my case I am using the React component and keep a ref to the typewriter object.  Setting the eventQueue to empty array does reset the queue. I'm sure there is something wrong with what I'm doing but for now its fine :)

**Comment by kmt29** [comment]
I never thought of but this should work too....Just a workaround tho....
